foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/queries/ReportItem.java

```diff
@@ -130,21 +131,16 @@ public void initialize(ReportQuery query) throws QueryException {
         if (this.mapping == null) {
             if (this.attributeExpression != null) {
                 DatabaseMapping mapping = this.attributeExpression.getLeafMapping(query, query.getDescriptor(), query.getSession());
-                if (mapping == null && this.attributeExpression.isFunctionExpression() && this.getResultType() == null){
-                    FunctionExpression expression = ((FunctionExpression)this.attributeExpression);
-                    if (expression.getOperator().equals(ExpressionOperator.maximum()) || expression.getOperator().equals(ExpressionOperator.minimum())){
-                        mapping = expression.getBaseExpression().getLeafMapping(query, query.getDescriptor(), query.getSession());
-                    }
-                }
-                if (mapping == null){
+                if (mapping == null) {
+                    //TODO: This feels like it should be set earlier during ReportItemBuild.visit. Should investigate.
```

Comment:
Spotted a TODO, did you intend to keep that in here, or investigate the issue?

Reply:
I intended to leave that one there. I was running through this code section so much in my testing that I started to think that this other mapping check might be a poorly coded section that should have been fixed somewhere else (similar to the code right before that I altered). But it wasnt related to what I was currently fixing so I wanted to comment on it and not lose track of a possible issue there.